operations/validate.go

```diff
@@ -29,13 +29,25 @@ func Validate() cli.Command {
 		}, cli.StringSliceFlag{
 			Name:  joinFlagNames(localModulesFlagName, "lm"),
 			Usage: "specify local modules as MODULE_NAME=PATH pairs",
+		}, cli.StringFlag{
+			Name:  joinFlagNames(projectFlagName, "p"),
+			Usage: "specify project identifier in order to run validation requiring project settings",
 		}),
 		Before: mergeBeforeFuncs(setPlainLogger, requirePathFlag),
 		Action: func(c *cli.Context) error {
 			confPath := c.Parent().String(confFlagName)
 			path := c.String(pathFlagName)
 			quiet := c.Bool(quietFlagName)
 			long := c.Bool(longFlagName)
+			projectID := c.String(projectFlagName)
+			if projectID == "" {
+				cwd, err := os.Getwd()
+				grip.Error(errors.Wrap(err, "unable to get current working directory"))
+				cwd, err = filepath.EvalSymlinks(cwd)
+				grip.Error(errors.Wrap(err, "unable to resolve symlinks"))
+				conf := ClientSettings{}
```

Comment:
This is initializing an empty client settings which will not be able to find the file.  Use the conf variable defined on line 59 (conf, err := NewClientSettings(confPath))